csireverseproxy/pkg/standaloneproxy/standaloneproxy.go

```diff
@@ -109,7 +107,7 @@ func newReverseProxy(mgmtServer config.ManagementServer) common.Proxy {
 func newTLSConfig(mgmtServer config.ManagementServer) *tls.Config {
 	// #nosec G402
 	tlsConfig := tls.Config{
-		InsecureSkipVerify: mgmtServer.SkipCertificateValidation,
+		InsecureSkipVerify: mgmtServer.SkipCertificateValidation, // #nosec,G402 - InsecureSkipVerify needs to be configurable
```

Comment:
It is configurable.  Need a better explanation here

Reply:
Updated comment